ups/verify.table

```diff
@@ -1,4 +1,8 @@
-setupRequired(sconsUtils)
+setupRequired(astropy)
+setupRequired(pyyaml)
+setupRequired(requests)
+setupRequired(numpy)
+setupRequired(base)
```

Comment:
utils is bringing in base, and your `__init__.py` files go out of their way to work even if `base` is missing, so I'd tend to not put `base` in here.